packages/graphql-codegen-cli/src/hooks.ts

```diff
@@ -74,10 +75,11 @@ async function executeHooks(
 ): Promise<void> {
   debugLog(`Running lifecycle hook "${hookName}" scripts...`);
 
+  const quotedArgs = quote(args);
   for (const script of scripts) {
     if (typeof script === 'string') {
-      debugLog(`Running lifecycle hook "${hookName}" script: ${script} with args: ${args.join(' ')}...`);
-      await execShellCommand(`${script} ${args.join(' ')}`);
+      debugLog(`Running lifecycle hook "${hookName}" script: ${script} with args: ${quotedArgs}...`);
+      await execShellCommand(`${script} ${quotedArgs}`);
     } else {
       debugLog(`Running lifecycle hook "${hookName}" script: ${script.name} with args: ${args.join(' ')}...`);
```

Comment:
`quotedArgs` is not used here on purpose as the text in the log message would contain different values than `script` receives. It could be confusing to have escaped arguments in the log but not in the function.